socioeconomic-differences/socioeconomic-differences.md

```diff
@@ -1,7 +1,7 @@
 # Socioeconomic-differences
 
 ### Projected Time
-~30
```

Comment:
Please reference this outline and format the projected time and headers that way: https://github.com/Techtonica/curriculum/blob/master/algorithms/intro-to-algorithms.md